src/ceph-daemon

```diff
@@ -0,0 +1,1290 @@
+#!/usr/bin/env python3
+
+DEFAULT_IMAGE='ceph/daemon-base'
+DATA_DIR='/var/lib/ceph'
+LOG_DIR='/var/log/ceph'
+UNIT_DIR='/etc/systemd/system'
+VERSION='unknown development version'
```

Comment:
VERSION is unused (running pyflakes on this file might find other things like this)

Reply:
fixing